site/content/blog/2022-10-01-whats-new-in-svelte-october-2022.md

```diff
@@ -0,0 +1,136 @@
+- [5:37:06](https://www.youtube.com/watch?v=A8jkJTWacow&t=20226s) - When Keeping it Svelte Goes Wrong. An Analysis of Some of the Worst Svelte I Have Ever Coded
+- [7:22:05](https://www.youtube.com/watch?v=A8jkJTWacow&t=26525s) - Gettng started with Hooks
+- [7:38:14](https://www.youtube.com/watch?v=A8jkJTWacow&t=27494s) - TBA*
+
+*In the final talk of the summit, Rich Harris shares the path to the next step for SvelteKit... the first Release Candidate! With no planned breaking changes left for kit, the team is hard at work squashing bugs and adding the remaining features for 1.0...
+
+## More SvelteKit Updates
+- `use:enhance` is the easiest way to progressively enhance a form ([Docs](https://kit.svelte.dev/docs/form-actions#progressive-enhancement-use-enhance), [#6633](https://github.com/sveltejs/kit/pull/6633), [#6828](https://github.com/sveltejs/kit/pull/6828), [#7012](https://github.com/sveltejs/kit/pull/7012))
+- Support for `$env/dynamic/*` allows SvelteKit to support Vite ecosystem tools such as Vitest and `vite-node` ([#6454](https://github.com/sveltejs/kit/pull/6454))
+- `+server.js` files can now be made next to `+page` files ([Docs](https://kit.svelte.dev/docs/routing#server-receiving-data), [#6773](https://github.com/sveltejs/kit/pull/6773))
+
+**Breaking changes:**
+- Node 16.14 is now the minimum version to run SvelteKit ([#6388](https://github.com/sveltejs/kit/pull/6388))
+- `App.PrivateEnv` and `App.PublicEnv` have been removed in favour of generated types ([#6413](https://github.com/sveltejs/kit/pull/6413))
+- If you use error.html and customized it, replace `%sveltekit.message%`  with `%sveltekit.error.message%` ([6659](https://github.com/sveltejs/kit/pull/6659))
+- `App.PageError` is now `App.Error` - check for it in your hooks ([Docs](https://kit.svelte.dev/docs/hooks#shared-hooks-handleerror), [#6963](https://github.com/sveltejs/kit/pull/6963))
+
+For a full list of changes, check out kit's [CHANGELOG](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md).
+
+## Svelte Updates
+- New a11y warnings for `incorrect-aria-attribute-type`, `no-abstract-role`, `interactive-element-to-noninteractive-role` and `role-has-required-aria-props`.`no-noninteractive-tabindex` and `click-events-have-key-events` coming soon! (**3.50.0**)
+- New types for `ComponentEvents` and `SveltePreprocessor` (**3.50.0**)
+- Improved parsing speed when encountering large blocks of whitespace (**3.50.0**)
+- All global JavaScript objects and functions are now recognized as known globals (**3.50.1**)
+
+For all the changes to the Svelte compiler, including upcoming changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## New in Language Tools
+- Better handling of configs for code formatting (**106.0.0**, [#1598](https://github.com/sveltejs/language-tools/pull/1598))
```

Comment:
```suggestion
- Better code formatting for editor suggestion (**106.0.0**, [#1598](https://github.com/sveltejs/language-tools/pull/1598))
```

Or something along the line. This change is the formatting of the code suggestion like the auto import, light bulb quick fix and organize-import.

Reply:
This is much better! I was having a hard time understanding the full scope of it from the PR 😅